### PR TITLE
Items

### LINKED ISSUE #26: Creation of items
Items that could be to personalize the character (badges for example), to gain bonus in duels, to gain ability to fake the presence in course, etc... 
Conceptually and in the Player class + a item interface and class

- [x] #164
- [x] #163

### LINKED ISSUE #163: Creation of items class, integration with remote database, association with students

### LINKED ISSUE #164: Add activity to display users items

### PR DESCRIPTION
**Pull request by Gorzen**
fixes #26 
fixes #164 
fixes #163

### CODE REVIEW

**Comment by nheyek** [comment]
Nice coverage

File: app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -62,14 +60,15 @@
...
+            android:label="@string/title_activity_maps" />
 
         <service
             android:name=".map.BackgroundLocation"
             android:exported="true"
             android:permission="android.permission.BIND_JOB_SERVICE" />
 
-        <activity android:name=".questions.DisplayQuestionActivity"/>
+        <activity android:name=".questions.DisplayQuestionActivity" />
+        <activity android:name=".items.DisplayItemActivity"></activity>
```
Comment (JulesssG) [inline app/src/main/AndroidManifest.xml]: Why does the activity deosnt have the same syntax as the other ones? (/> becomes ></activity>) Is it the same?

File: app/src/androidTest/java/ch/epfl/sweng/studyup/UtilsTest.java [hunk 2]
```diff
@@ -0,0 +1,52 @@
+package ch.epfl.sweng.studyup;
```
Comment (JulesssG) [inline app/src/androidTest/java/ch/epfl/sweng/studyup/UtilsTest.java]: Aren't those some UnitTest and not intrumentationTests?

File: app/src/main/java/ch/epfl/sweng/studyup/items/DisplayItemActivity.java [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+import android.os.Bundle;
+import android.view.View;
+import android.widget.Button;
+import android.widget.ImageView;
+import android.widget.TextView;
+
+import ch.epfl.sweng.studyup.R;
+import ch.epfl.sweng.studyup.player.Player;
+
+public class DisplayItemActivity extends AppCompatActivity {
```
Comment (JulesssG) [inline app/src/main/java/ch/epfl/sweng/studyup/items/DisplayItemActivity.java]: Shouldn't it extends RefreshContext to be consistent?

File: hs_err_pid10787.log [hunk 4]
```diff
@@ -0,0 +1,112 @@
+#
```
Comment (JulesssG) [inline hs_err_pid10787.log]: Shouldnt this file be ignored?

File: app/src/main/AndroidManifest.xml (hunk 1 above)
Comment (Gorzen) [reply app/src/main/AndroidManifest.xml]: Yes, it's the same, just like html.

File: app/src/main/java/ch/epfl/sweng/studyup/items/DisplayItemActivity.java (hunk 3 above)
Comment (Gorzen) [reply app/src/main/java/ch/epfl/sweng/studyup/items/DisplayItemActivity.java]: Yes

File: app/src/androidTest/java/ch/epfl/sweng/studyup/UtilsTest.java (hunk 2 above)
Comment (Gorzen) [reply app/src/androidTest/java/ch/epfl/sweng/studyup/UtilsTest.java]: Yes, the same goes for ItemsTest

File: hs_err_pid10787.log (hunk 4 above)
Comment (Gorzen) [reply hs_err_pid10787.log]: Yes

**Pull request merged**